galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -6,6 +9,32 @@
 from galaxy_api.api.ui import serializers
 
 
+class NamespaceFilter(filterset.FilterSet):
+    keywords = filters.CharFilter(method='keywords_filter')
+
+    sort = filters.OrderingFilter(
+        fields=(
+            ('name', 'name'),
+            ('company', 'company'),
+            ('id', 'id'),
+        ),
+    )
+
+    class Meta:
+        model = models.Namespace
+        fields = ('name', 'company',)
+
+    def keywords_filter(self, queryset, name, value):
+
+        keywords = self.request.query_params.getlist('keywords')
+
+        if keywords is not None:
```

Comment:
```
for keyword in keywords or []
```

Reply:
yup, the None check is unneeded.

Updated.